### ISSUE TITLE
Play icon in the "getting started" dialog fails non-text contrast requirement

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#856]
The following AI-windows mode(s) have this issue:
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![3 1](https://user-images.githubusercontent.com/22068434/88342849-0d468280-ccf5-11ea-81f1-811920589363.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in release [1.1.1380.01](https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1380.01). Please regress and close at your convenience.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1380.01.
